parachain/primitives/beacon/src/ssz.rs

```diff
@@ -161,17 +178,17 @@ impl From<ExecutionPayloadHeader> for SSZExecutionPayloadHeader {
 			block_hash: payload.block_hash.to_fixed_bytes(),
 			transactions_root: payload.transactions_root.to_fixed_bytes(),
 			withdrawals_root: payload.withdrawals_root.to_fixed_bytes(),
-		}
+		})
 	}
 }
 
-pub fn hash_tree_root<T: SimpleSerialize>(mut object: T) -> Result<H256, MerkleizationError> {
+pub fn hash_tree_root<T: SimpleSerialize>(mut object: T) -> Result<H256, SimpleSerializeError> {
```

Comment:
Changed from `MerklizationError` to `SimpleSerializeError` since the error returned here could now also be a `DeserializeError`.